src/components/InstructionDisplay.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SimplePieChart from './SimplePieChart';
+import useCPU from '../hooks/use-cpu';
+
+const InstructionDisplay = () => {
+  const cpu = useCPU();
+
+  if (!cpu) {
+    return <h1>Loading...</h1>;
+  }
+  console.log(cpu.instruction_amounts);
```

Comment:
maybe add more descriptions to your logs or remove them before merging, so that we don't have see random numbers in the dev console

Reply:
done